var/spack/repos/builtin/packages/mark/package.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+import os
+
+
+class Mark(Package):
+    """Program MARK, developed and maintained by Gary White (Colorado State
+    University) is the most flexible, widely used application currently
+    available for parameter estimation using data from marked individuals.
+
+    You will need to download the package yourself, unzip, rename it
```

Comment:
:scream:

Reply:
This doesn't have to be the case.  I was able to work around it with an agent string passed to curl.  See https://github.com/chuckatkins/spack/commit/dbd40a42d9d7b0ab8101991a21e38fa4185f7f60